js/index.js

```diff
@@ -63,6 +63,41 @@ function displayProjects() {
   projectCards.appendChild(cards);
 }
 
+const handleSubmit = (event) => {
+  event.preventDefault();
+  const nameError = document.getElementById('name-error');
+  const emailError = document.getElementById('email-error');
+  const msgError = document.getElementById('msg-error');
+  const name = document.getElementById('name-input').value;
+  const email = document.getElementById('email-input').value;
+  const msg = document.getElementById('msg-input').value;
+  let error = false;
+  if (name.length === 0) {
+    nameError.innerHTML = 'Name is required';
+    error = true;
+  }
+  if (email.length === 0) {
+    emailError.innerHTML = 'Email is required';
+    error = true;
```

Comment:
- [ ] The error is still there kindly fix it.

Reply:
please check the line bellow